chain/txValidator.go

```diff
@@ -265,7 +271,7 @@ func CheckTransactionPayload(txn *transaction.Transaction, height uint32) error
 }
 
 // VerifyTransactionWithLedger verifys a transaction with history transaction in ledger
-func VerifyTransactionWithLedger(txn *transaction.Transaction) error {
+func VerifyTransactionWithLedger(txn *transaction.Transaction, height uint32) error {
```

Comment:
We should remove line `height := DefaultLedger.Store.GetHeight()` in VerifyTransactionWithLedger because height is now passed in as argument